Modules/mathmodule.c

```diff
@@ -2163,6 +2163,27 @@ math_ldexp_impl(PyObject *module, double x, PyObject *i)
     } else {
         errno = 0;
         r = ldexp(x, (int)exp);
+#ifdef _MSC_VER
+        if (-DBL_MIN < r && r < DBL_MIN) {
```

Comment:
```suggestion
        if (DBL_MIN > r && r > -DBL_MIN) {
```
I'm not sure I can believe in that bias.  Though, suggestion seems not worse than the current variant - and I'll accept that.